content/5_Plat/Centroid.mdx

```diff
@@ -190,43 +192,51 @@ subgraphs.
 
 General centroid code is shown below.
 
-<IncompleteSection>
-
-Ben - this is not easy to understand :/
-
-</IncompleteSection>
-
 <LanguageSection>
 
 <CPPSection>
 
 <!-- pulled from https://codeforces.com/contest/1303/submission/76216413, which I think is my most recent centroid problem -->
 
 ```cpp
-bool r[MN];  // removed
-int s[MN];   // subtree size
-int dfs(int n, int p = 0) {
-	s[n] = 1;
-	for (int x : a[n])
-		if (x != p && !r[x]) s[n] += dfs(x, n);
-	return s[n];
+vector<int> adj[maxn];  // adjacency list
+bool r[MN];             // removed
+int s[MN];              // subtree size
```

Comment:
hey man
how about comments
you just...make the variable names themselves more intuitive